integrations/extensions/docs/ADVANCED_USAGE.md

```diff
@@ -117,4 +117,6 @@ For illustration purposes, let's say a customer wanted to be able to link their
 1. User selects which ticket ID to add to Zendesk ticket.
 1. The selected HubSpot ticket information is added into Zendesk ticket as a comment.
 
+For better illustration on how to implement something like this, a MetaWeather and Zendesk skill is included and can be found [here](../starter-kits/metaweather/readme.md#even-more-advanced-using-the-metaweather-extension-with-another-extension) in the starter kit. 
```

Comment:
I would vote to remove the hubspot-> zendsesk  hand-waving example and use the example u just added

Reply:
You're so right lol